src/ray/object_manager/object_manager.cc

```diff
@@ -211,12 +220,13 @@ ray::Status ObjectManager::PullSendRequest(const ObjectID &object_id,
   auto message = object_manager_protocol::CreatePullRequestMessage(
       fbb, fbb.CreateString(client_id_.binary()), fbb.CreateString(object_id.binary()));
   fbb.Finish(message);
-  RAY_CHECK_OK(conn->WriteMessage(
+  Status status = conn->WriteMessage(
       static_cast<int64_t>(object_manager_protocol::MessageType::PullRequest),
-      fbb.GetSize(), fbb.GetBufferPointer()));
-  RAY_CHECK_OK(
-      connection_pool_.ReleaseSender(ConnectionPool::ConnectionType::MESSAGE, conn));
-  return ray::Status::OK();
+      fbb.GetSize(), fbb.GetBufferPointer());
+  if (status.ok()) {
+    connection_pool_.ReleaseSender(ConnectionPool::ConnectionType::MESSAGE, conn);
```

Comment:
Return code shouldn't be ignored here

Reply:
Oops, I meant to remove the return value. I'll do that now, thanks!